.circleci/config.yml

```diff
@@ -74,6 +74,10 @@ commands:
           name: Run Rust sample
           command: |
             cargo run -p sample
+      - run:
+          name: Run Rust RLB test
+          command: |
+            bin/rust-external-test.sh
```

Comment:
Nit: maybe we name this something more tightly connected to what it's testing? It isn't testing RLB wholesale, it's "just" running the `long-running` example and checking if the pending ping dir is empty.

Or do you have plans for this becoming a test suite of its own, so the generic naming is future-looking?

Reply:
I ... initially thought about making this more generic, but given it's only one test right now naming it more appropriately is in order.